### ISSUE TITLE
Hydra canvas magnifying filter not working as NEAREST in some browsers (qutebrowser)

### ISSUE THREAD

**Issue by ritchse** [ojack/hydra#145]
Setting Hydra's canvas magnifying filter
---
### Bug
Lately I've been using hydra in very low-res in my old laptop, and experimenting with that pixelated almost retro look. However, I switched from Chromium to Qutebrowser yesterday and realized that it didn't look pixelated at all, it looked smoothed out like a video in 240p. I couldn't find a solution but I suspect it maaay have to do with [setting the parameter `TEXTURE_MAG_FILTER` to `NEAREST` instead of the default `LINEAR`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/getTexParameter).
I saw in the css of the `hydra-canvas` that `image-rendering` is set to `pixelated`, which seems to work on most browsers but not on qutebrowser for some reason? It may have to do with it being a web-kit-based browser.
### Request
However, I thought the linear interpolation looked cool with some sketches (specially the ones that involve the camera, it gave them something of a 2006 youtube look or even a VHS looks in certain scenarios). That's why I think, even if this is more of a bug, **what I want to express in this issue is the request to be able to freely change the interpolation filter/method used by Hydra, be it dynamically (inside the editor itself) or not (maybe an argument in the URL?).**

**Comment by ojack** [comment]
Yes, I think the pixelation is coming from the css property rather than the TEXTURE_MAG_FILTER. I have not played around with the different filtering modes very much. Would you want to change them on all sources and outputs at once, or be able to control each one separately?

**Comment by ritchse** [comment]
I didn't know the latter was possible! But I meant all outputs at once

**Comment by ojack** [comment]
Here is where it is defined in the code for each output: https://github.com/ojack/hydra-synth/blob/67741ed9f098ed7c6a8aea2a37857d7fd7e64bef/src/output.js#L20

I can't remember why I set it to NEAREST but I think it was in order to render pixelated images.
